fv/wireguard_test.go

```diff
@@ -820,6 +824,37 @@ var _ = infrastructure.DatastoreDescribe("WireGuard-Supported 3 node cluster", [
 				}, "10s", "100ms").Should(BeNumerically(">", 0))
 			}
 		})
+
+		It("WireGuard should be used for host to workload connections on WireGuard enabled nodes", func() {
+			cc.ExpectSome(felixes[0], wls[1])
+			cc.CheckConnectivity()
+
+			By("verifying packets between felix-0 and felix-1 is encrypted")
+			for i := range []int{0, 1} {
+				Eventually(func() int {
+					return tcpdumps[i].MatchCount("numTunnelPackets01")
+				}, "10s", "100ms").Should(BeNumerically(">", 0))
+				Eventually(func() int {
+					return tcpdumps[i].MatchCount("numTunnelPackets10")
+				}, "10s", "100ms").Should(BeNumerically(">", 0))
+			}
+		})
+
+		It("WireGuard should not be used for host to workload connections when WireGuard disabled on either node", func() {
+			cc.ExpectSome(felixes[0], wls[2])
+			cc.ExpectSome(felixes[2], wls[0])
+			cc.CheckConnectivity()
+
+			By("verifying packets between felix-0 and felix-2 are not encrypted")
+			for i := range []int{0, 2} {
```

Comment:
Should this be?

`for _, i := range []int{0, 2} {`

Reply:
Thanks, good catch!